packages/blockchain-wallet-v4-frontend/src/scenes/Interest/InterestInfo/index.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Button,
+  Icon,
+  Link,
+  SkeletonRectangle,
+  Text
+} from 'blockchain-info-components'
+import { CustomBox } from 'components/Layout'
+import { FormattedMessage } from 'react-intl'
+import { Props, State } from '..'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const ContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+class EarnInterestInfo extends PureComponent<Props & State> {
+  render () {
+    return (
+      <CustomBox>
+        <div>
+          <Icon name='savings-icon' color='blue600' size='32px' />
+          <Text
+            size='20px'
+            color='grey800'
+            weight={600}
+            style={{ marginTop: '16px' }}
+          >
+            <FormattedMessage
+              id='scenes.earninterest.earnheader'
+              defaultMessage='Earn Crypto from Your Blockchain Wallet'
+            />
+          </Text>
+        </div>
+        {this.props.isDisabled ? (
+          this.props.userDataR.cata({
+            Success: val => (
+              <ContentWrapper>
+                <Text
+                  size='14px'
+                  color='grey600'
+                  weight={500}
+                  style={{ marginTop: '10px', lineHeight: 1.5 }}
+                >
+                  <FormattedMessage
+                    id='scenes.earninterest.earnbody.unverified'
+                    defaultMessage='Upgrade to Gold Level and be eligible to earn up to 3% on your crypto.'
+                  />
+                </Text>
+                <Button
+                  nature='primary'
+                  data-e2e='verifyIdentityBorrow'
+                  style={{ marginTop: '20px' }}
+                  disabled={val.kycState !== 'NONE'}
+                  onClick={() =>
+                    this.props.identityVerificationActions.verifyIdentity(2)
+                  }
+                >
+                  {val.kycState === ('UNDER_REVIEW' || 'PENDING') ? (
```

Comment:
This syntax won't actually work the way you want it to.

The reason for this is that `('UNDER_REVIEW' || 'PENDING')` _always_ evaluates to `UNDER_REVIEW`

Reply:
```
kycState = 'PENDING'
=> "PENDING"
kycState === ('UNDER_REVIEW' || 'PENDING')
=> false
```